### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sjib**
Added new template feature request

### CODE REVIEW

**Comment by sjib** [comment]
@ponceta I made a new template for new feature requests. Is this still in line with the template you have? Else pleaes modify,

**Comment by sjib** [comment]
ping @kandre @ponceta I made a new template for new feature requests. Is this still in line with the template you have? Else pleaes modify,

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+- Tender reception date : 
+- Tender validation date : 
+
+## Documentation
+
+Documentation links
+- User manual : 
+- Administrator manual :
+
+## Development and intégration
```
Comment (ponceta) [inline .github/ISSUE_TEMPLATE/feature_request.md:97]:
```suggestion
## Development and integration
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- Database model version: 
+- INTERLIS Model version: 
+
+Public issues links : 
+- …
+
+Helpdesk issues links : 
+- …
+
+### Dates de réalisation
```
Comment (ponceta) [inline .github/ISSUE_TEMPLATE/feature_request.md:86]:
```suggestion
### Realisation dates
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+_Add a detailed timeline_
+
+- [x] 01.09.2021 : First draft
+- [ ] 01.10.2021 : 
+- [ ] 01.11.2021 : 
+
+
+- [ ] Warranty period : to be adressed
+
+### Versions of related softwares
```
Comment (ponceta) [inline .github/ISSUE_TEMPLATE/feature_request.md:74]:
```suggestion
### Related softwares versions
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+- Future prospects : 
+
+## Conclusion
+
+Conclusions from applicant, users or admins group
+
+- Conclusion: 
+- Future prospects:
+
+- Invoice date:
```
Comment (ponceta) [inline .github/ISSUE_TEMPLATE/feature_request.md:112]: Not shared on Github
```suggestion
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+## Conclusion
+
+Conclusions from applicant, users or admins group
+
+- Conclusion: 
+- Future prospects:
+
+- Invoice date: 
+- Payment date:
```
Comment (ponceta) [inline .github/ISSUE_TEMPLATE/feature_request.md:113]: Not shared on Github
```suggestion
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+
+### Risks and constraints
+
+
+### Mutualisation 
+_Mutualisation and possible impacts with other projects (QGIS, QWAT, QGEP)_
+
+
+## Cahier des charges et offres ( Developer)
```
Comment (ponceta) [inline .github/ISSUE_TEMPLATE/feature_request.md:55]:
```suggestion
## Specification and offers ( Developer )
```

**Review by ponceta — CHANGES_REQUESTED** [review]
Thank you Stefan, this is nice, I let you check my comments.

I let you merge this and I'll open some current feature requests as examples to check if this applies well.

**Review by sjib — COMMENTED** [review]
ok now?

**Review by sjib — COMMENTED** [review]
@ponceta I have added the changes. Can you merge?

**Comment by ponceta** [comment]
@sjib Seems fine to me!

**Review by ponceta — APPROVED** [review]

**Pull request merged**